app/components/chat_component.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class='active-admin-chat__conversation-history-container'>
+  <div class='active-admin-chat__conversation-history' data-motion='scroll->load_messages'>
+    <% if conversation && messages.empty? %>
+        <div class='active-admin-chat__conversation-history no-messages'>No messages</div>
+    <% else %>
+      <%= render partial: 'active_admin/chat/messages', locals: { messages: messages } %>
+    <% end %>
+  </div>
+  <div class='active-admin-chat__send-message-container'>
+    <form class='active-admin-chat__send-message-form' data-motion='send_message'>
+      <input id='<%= send_message_id %>' type='text' name='message' placeholder='Send message' />
```

Comment:
I set a random `id` so that I get a fresh new empty input when a message is sent